src/locator/al-locator.types.ts

```diff
@@ -92,15 +101,16 @@ export class AlLocation
                 aliases: [
                     `https://${appCode}.ui-dev.product.dev.alertlogic.com`,
                     `https://${appCode}-*.ui-dev.product.dev.alertlogic.com`,
-                    `https://${appCode}-*-*.ui-dev.product.dev.alertlogic.com`,
-                    `https://${appCode}-*-*-*.ui-dev.product.dev.alertlogic.com`,
```

Comment:
removing these will mean that we cant have 
app-my-demo.ui-dev.product.dev.alertlogic.com
or could we add the - to the regex?

Reply:
I'll add `-` to the regex expansion.